<ui version="4.0">
<class>Editpreaut</class>
<widget class="QDialog" name="Editpreaut">
 <property name="geometry">
  <rect>
   <x>0</x>
   <y>0</y>
   <width>800</width>
   <height>600</height>
  </rect>
 </property>
 <property name="windowTitle">
  <string>Edit</string>
 </property>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>781</width>
       <height>500</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
 <widget class="QCheckBox" name="ChkBxsunday">
<property name="geometry">
 <rect>
  <x>100</x>
  <y>5</y>
  <width>110</width>
  <height>22</height>
 </rect>
</property>
<property name="text">
 <string>sunday</string>
</property>
</widget>
<widget class="QCheckBox" name="ChkBxmonday">
<property name="geometry">
 <rect>
  <x>100</x>
  <y>40</y>
  <width>110</width>
  <height>22</height>
 </rect>
</property>
<property name="text">
 <string>monday</string>
</property>
</widget>
<widget class="QCheckBox" name="ChkBxtuesday">
<property name="geometry">
 <rect>
  <x>100</x>
  <y>75</y>
  <width>110</width>
  <height>22</height>
 </rect>
</property>
<property name="text">
 <string>tuesday</string>
</property>
</widget>
<widget class="QCheckBox" name="ChkBxwednesday">
<property name="geometry">
 <rect>
  <x>100</x>
  <y>110</y>
  <width>110</width>
  <height>22</height>
 </rect>
</property>
<property name="text">
 <string>wednesday</string>
</property>
</widget>
<widget class="QCheckBox" name="ChkBxthursday">
<property name="geometry">
 <rect>
  <x>100</x>
  <y>145</y>
  <width>110</width>
  <height>22</height>
 </rect>
</property>
<property name="text">
 <string>thursday</string>
</property>
</widget>
<widget class="QCheckBox" name="ChkBxfriday">
<property name="geometry">
 <rect>
  <x>100</x>
  <y>180</y>
  <width>110</width>
  <height>22</height>
 </rect>
</property>
<property name="text">
 <string>friday</string>
</property>
</widget>
<widget class="QCheckBox" name="ChkBxsaturday">
<property name="geometry">
 <rect>
  <x>100</x>
  <y>215</y>
  <width>110</width>
  <height>22</height>
 </rect>
</property>
<property name="text">
 <string>saturday</string>
</property>
</widget>
<widget class="QLabel" name="Lblvisit">
<property name="geometry">
 <rect>
  <x>30</x>
  <y>255</y>
  <width>59</width>
  <height>17</height>
 </rect>
</property>
<property name="text">
 <string>visit</string>
</property>
</widget>
<widget class="SComboBox" name="CmbBxvisit">
<property name="geometry">
 <rect>
  <x>100</x>
  <y>250</y>
  <width>120</width>
  <height>33</height>
 </rect>
</property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QComboBox::AdjustToContents</enum>
    </property>
<item>
<property name="text">
 <string></string>
</property>
</item>
</widget>
<widget class="QLabel" name="Lblhoraini">
<property name="geometry">
 <rect>
  <x>30</x>
  <y>290</y>
  <width>59</width>
  <height>17</height>
 </rect>
</property>
<property name="text">
 <string>horaini</string>
</property>
</widget>
<widget class="QTimeEdit" name="TmEdthoraini">
<property name="geometry">
 <rect>
  <x>100</x>
  <y>285</y>
  <width>118</width>
  <height>27</height>
 </rect>
</property>
</widget>
<widget class="QLabel" name="Lblhorafim">
<property name="geometry">
 <rect>
  <x>30</x>
  <y>325</y>
  <width>59</width>
  <height>17</height>
 </rect>
</property>
<property name="text">
 <string>horafim</string>
</property>
</widget>
<widget class="QTimeEdit" name="TmEdthorafim">
<property name="geometry">
 <rect>
  <x>100</x>
  <y>320</y>
  <width>118</width>
  <height>27</height>
 </rect>
</property>
</widget>
<widget class="QLabel" name="Lblvalidate">
<property name="geometry">
 <rect>
  <x>30</x>
  <y>360</y>
  <width>59</width>
  <height>17</height>
 </rect>
</property>
<property name="text">
 <string>validate</string>
</property>
</widget>
<widget class="QDateEdit" name="DtEdtvalidate">
<property name="geometry">
 <rect>
  <x>100</x>
  <y>355</y>
  <width>110</width>
  <height>27</height>
 </rect>
</property>
</widget>
<widget class="QLabel" name="Lblauthorizer">
<property name="geometry">
 <rect>
  <x>30</x>
  <y>395</y>
  <width>59</width>
  <height>17</height>
 </rect>
</property>
<property name="text">
 <string>authorizer</string>
</property>
</widget>
<widget class="SComboBox" name="CmbBxauthorizer">
<property name="geometry">
 <rect>
  <x>100</x>
  <y>390</y>
  <width>120</width>
  <height>33</height>
 </rect>
</property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QComboBox::AdjustToContents</enum>
    </property>
<item>
<property name="text">
 <string></string>
</property>
</item>
</widget>
<widget class="QLabel" name="Lblobs">
<property name="geometry">
 <rect>
  <x>30</x>
  <y>430</y>
  <width>59</width>
  <height>17</height>
 </rect>
</property>
<property name="text">
 <string>obs</string>
</property>
</widget>
<widget class="QLineEdit" name="LnEdtobs">
 <property name="geometry">
  <rect>
   <x>100</x>
   <y>425</y>
   <width>113</width>
   <height>27</height>
  </rect>
 </property>
</widget>
<widget class="QLabel" name="Lblautsince">
<property name="geometry">
 <rect>
  <x>30</x>
  <y>465</y>
  <width>59</width>
  <height>17</height>
 </rect>
</property>
<property name="text">
 <string>autsince</string>
</property>
</widget>
<widget class="QDateEdit" name="DtEdtautsince">
<property name="geometry">
 <rect>
  <x>100</x>
  <y>460</y>
  <width>110</width>
  <height>27</height>
 </rect>
</property>
</widget>
<widget class="QLabel" name="Lblreason">
<property name="geometry">
 <rect>
  <x>30</x>
  <y>500</y>
  <width>59</width>
  <height>17</height>
 </rect>
</property>
<property name="text">
 <string>reason</string>
</property>
</widget>
<widget class="SComboBox" name="CmbBxreason">
<property name="geometry">
 <rect>
  <x>100</x>
  <y>495</y>
  <width>120</width>
  <height>33</height>
 </rect>
</property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QComboBox::AdjustToContents</enum>
    </property>
<item>
<property name="text">
 <string></string>
</property>
</item>
</widget>
    </widget>
<widget class="QFrame" name="topmenu">
 <property name="geometry">
  <rect>
   <x>1</x>
   <y>1</y>
   <width>801</width>
   <height>42</height>
  </rect>
 </property>
 <property name="minimumSize">
  <size>
   <width>0</width>
   <height>42</height>
  </size>
 </property>
 <property name="maximumSize">
  <size>
   <width>16777215</width>
   <height>42</height>
  </size>
 </property>
 <property name="styleSheet">
  <string notr="true">background-color: rgb(50, 0, 133);</string>
 </property>
 <property name="frameShadow">
  <enum>QFrame::Raised</enum>
 </property>
 <widget class="QLabel" name="labelStatus">
  <property name="enabled">
   <bool>false</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>4</x>
    <y>0</y>
    <width>541</width>
    <height>39</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>50</red>
        <green>0</green>
        <blue>133</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>50</red>
        <green>0</green>
        <blue>133</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>50</red>
        <green>0</green>
        <blue>133</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>50</red>
        <green>0</green>
        <blue>133</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>50</red>
        <green>0</green>
        <blue>133</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>50</red>
        <green>0</green>
        <blue>133</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>50</red>
        <green>0</green>
        <blue>133</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>50</red>
        <green>0</green>
        <blue>133</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>50</red>
        <green>0</green>
        <blue>133</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>Verdana</family>
    <pointsize>18</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="styleSheet">
   <string/>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Plain</enum>
  </property>
  <property name="lineWidth">
   <number>1</number>
  </property>
  <property name="text">
   <string>Cadastro de </string>
  </property>
  <property name="textFormat">
   <enum>Qt::AutoText</enum>
  </property>
  <property name="alignment">
   <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
  </property>
  <property name="wordWrap">
   <bool>false</bool>
  </property>
  <property name="textInteractionFlags">
   <set>Qt::NoTextInteraction</set>
  </property>
 </widget>
</widget>
<widget class="QPushButton" name="PshBtnCancel">
<property name="geometry">
 <rect>
  <x>622</x>
  <y>568</y>
  <width>85</width>
  <height>27</height>
 </rect>
</property>
<property name="text">
 <string>&amp;Cancelar</string>
</property>
</widget>
<widget class="QPushButton" name="PshBtnSave">
<property name="geometry">
 <rect>
  <x>709</x>
  <y>568</y>
  <width>85</width>
  <height>27</height>
 </rect>
</property>
<property name="text">
 <string>&amp;Salvar</string>
</property>
</widget>

</widget>
    <customwidgets>
     <customwidget>
      <class>SComboBox</class>
      <extends>QComboBox</extends>
      <header location="global">scombobox.h</header>
     </customwidget>
    </customwidgets>
<resources/>
<connections/>
</ui>
